app/controllers/saml_sessions_controller.rb

```diff
@@ -17,11 +17,8 @@ def destroy
     session["signed_out"] = true
     session["feedback_return_path"] = destroy_provider_session_path
     sign_out current_provider
-    if IdPSettingsAdapter.mock_saml?
-      redirect_to providers_root_url
-    else
-      redirect_to new_feedback_path
-    end
```

Comment:
I can't think of a reason why we shouldn't be redirecting all providers to the feedback page in all environments, so I've removed this extra complexity.

The commit history didn't convince me that this was something worth keeping, but someone might have some context that was missing.

Reply:
Have you tried it out and seen odd behaviour? 

This is something i have seen in other LAA/MoJ apps too. One reason might be convenience for devs/testers, another might be to prevent/avoid submission of feedback from non-prod environments which gets mixed in with real feedback 🤷 

I am happy to remove however!

Reply:
I'm not sure what you mean by "odd behaviour"?

As I said, the commit history doesn't indicate that this distinction is important —so I removed it.

In the interests of brevity and not boring people to death, I omitted the details of the commit history (everyone can look if they need to).

I didn't expect this to be a particularly controversial change, but here is what I gleaned from the commit history to inform my decision.

---

In #185 we redirected to the `providers_root_url`, then in #418 we took the decision to log providers out of the portal when they logged out of our application (in production environments).

In #1193 we started redirecting providers to the feedback page instead when they logged out.

Before, the conditional was logical because it made no sense to log users out of the portal if they weren't logged in that way (i.e `mock_saml` is true).

As soon as we're no longer doing that (i.e we're just redirecting to the feedback page), the conditional adds no value.